packages/flutter_tools/lib/src/test/runner.dart

```diff
@@ -89,14 +125,65 @@ Future<int> runTests(
 
   // Call package:test's main method in the appropriate directory.
   final Directory saved = fs.currentDirectory;
+  
+  final Function printReloadMessage = () {
+    printStatus(
+      "Press 'r' to hot reload your tests, 'q' to quit",
```

Comment:
Instead of hot reload call this rerunning, it is more descriptive.

Reply:
Done